scientific_library/tvb/simulator/simulator.py

```diff
@@ -363,6 +373,16 @@ def _loop_monitor_output(self, step, state):
         if any(outputi is not None for outputi in output):
             return output
 
+    def bound_and_clamp(self, state):
```

Comment:
Could this clamping of the history values be separated and introduced into tvb-library as a separate PR with dedicated unit-tests ?
I think it is related to #13 and #6 Isn't it ?